AlphaWallet/Tokens/Types/TokenObject.swift

```diff
@@ -8,30 +8,34 @@ extension Activity {
 
     struct AssignedToken: Equatable, Hashable {
 
-        struct TokenBalance {
-            var balance = "0"
-            var json: String = "{}"
-        }
-
         enum Balance {
-            case value(NSDecimalNumber?)
```

Comment:
This seems like an unrelated change to the image size changes. Can you help to separate them?